util/src/main/java/tech/pegasys/artemis/util/mikuli/PublicKey.java

```diff
@@ -92,14 +101,19 @@ PublicKey combine(PublicKey pk) {
    *
    * @return byte array representation of the public key
    */
-  public Bytes toBytes() {
+  public Bytes toBytesCompressed() {
     return point.toBytesCompressed();
   }
 
   G1Point g1Point() {
     return point;
   }
 
+  @Override
+  public String toString() {
+    return "Signature [ecpPoint=" + point.toString() + "]";
+  }
+
   @Override
   public int hashCode() {
     final int prime = 31;
```

Comment:
Also not necessarily part of this diff, but I'd prefer we use the `Objects.hash(arg1, arg2, ...., argN)` method of generating a hash code here, as opposed to the older prime way of doing it.

Reply:
Done.